src/Tribe/Integrations/Manager.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__Manager
+ *
+ * Loads and manages the third-party plugins integration implementations.
+ */
+class Tribe__Events__Integrations__Manager {
+
+	/**
+	 * @var Tribe__Events__Integrations__Manager
+	 */
+	protected static $instance;
+
+	/**
+	 * The class singleton constructor.
+	 *
+	 * @return Tribe__Events__Integrations__Manager
+	 */
+	public static function instance() {
```

Comment:
That's totally fine, but we should probably avoid describing it as a singleton in that case - it's not really a singleton.